src/main/java/org/broadinstitute/hellbender/engine/dataflow/datasources/ReferenceShard.java

```diff
@@ -0,0 +1,90 @@
+package org.broadinstitute.hellbender.engine.dataflow.datasources;
+
+import com.google.cloud.dataflow.sdk.coders.*;
+import com.google.cloud.dataflow.sdk.values.KV;
+import htsjdk.samtools.util.Locatable;
+import org.broadinstitute.hellbender.utils.Utils;
+
+import java.io.Serializable;
+
+/**
+ * ReferenceShard is section of the reference genome that's used for sharding work for pairing things with
+ * the reference. The primary use case is pairing reads with the reference.
+ * This class is designed to be a simple data-storage class with related static utilities.
+ */
+public class ReferenceShard implements Serializable {
```

Comment:
can this be made final? non-final classes with equals and hashcode are problematic